src/styles/dgrid.css

```diff
@@ -1 +1,2 @@
+@import 'headerCell.css';
```

Comment:
if you're going to offer up a `dgrid.css` file for people using `dgrid` outside of our cli-webpack build then you need to ensure you add an exclusion rule for it in your gruntfile otherwise it will be built into a separate css-module and the generated classes will mean zero.